src/components/Markdown.svelte

```diff
@@ -0,0 +1,26 @@
+<script>
+  import { use, parse, parseInline } from 'marked';
+
+  const renderer = {
+    list(body, ordered, start) {
+      const outerEl = ordered ? 'ol' : 'ul';
+      return `<${outerEl} ${
+        start ? `start="${start}"` : ''
+      } class="mzp-u-list-styled">${body}</${outerEl}>`;
```

Comment:
I don't think this class applies for glam.

Reply:
you're right! Thanks for catching that!